### PR TITLE
Add a new project to the pipeline

### PR DESCRIPTION
**Pull request by isabelrios**
POC with changes needed to add a new projects.

This is a first version that could change depending on the needs / feedback

### CODE REVIEW

**Comment by isabelrios** [comment]
The only changes in database required are already implemented in staging:

New row in projects table:

> <div dir="ltr" style="margin-left:0pt;" align="left" id="docs-internal-guid-a3c1bff0-7fff-58ee-0707-5c303fb21c96">
> id | testrail_project_id | project_name_abbrev | project_name
> 
> <div dir="ltr" style="margin-left:0pt;" align="left" id="docs-internal-guid-b861f34e-7fff-989e-a2d5-135a065e7f2b">
> 6 | 54 | experimenter | Ecosystem QA - Experiments

> </div>


So that we have all the Test Suites for Project ID 54, correctly iterated and data stored in database for each one, it is necesary to fill in the custom field in test rail: **Automation Coverage** 

Otherwise the tests and their automation status is not counted.

File: __main__.py [hunk 1]
```diff
@@ -35,8 +41,28 @@ def parse_args(cmdln_args):
...
+            print("--project is required for the report selected")
+        if not platform:
+            print("--platform is required for the report selected")
+
+    if platform == 'mobile' and project not in PROJECTS_MOBILE:
+        print(f"Error: Invalid project '{project}' for mobile. Valid options are {PROJECTS_MOBILE}") # noqa 
+        sys.exit(1)
+    elif platform == 'ecosystem' and project not in PROJECTS_ECOSYSTEM:
+        print(f"Error: Invalid project '{project}' for ecosystem. Valid options are {PROJECTS_ECOSYSTEM}") # noqa
+        sys.exit(1)
```
Comment (b4handjr) [inline __main__.py:59]: This is a nice way to filter, similar to what I had in mind!

**Review by b4handjr — APPROVED** [review]
I think this looks good. A simple filter that doesn't involve too much code. Nice job!

**Review by rpappalax — APPROVED** [review]
Looking good, Isabel!  LGTM

**Pull request merged**